docs/application-connector/03-08-application-gateway-proxying-requests.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Proxying requests by the Application Gateway
+type: Details
+---
+
+The Application Gateway proxies requests from lambda functions and services in Kyma to external APIs. When the Application Gateway works in the alternative `gatewayOncePerNamespace` [mode](#architecture-application-connector-components-application-operator), it proxies the requests based on the configuration stored in Secrets.
```

Comment:
`gatewayOncePerNamespace` -> shouldn't we use the **bold** formatting here, just as we do with parameters?